Lib/test/test_embed.py

```diff
@@ -433,11 +433,12 @@ class InitConfigTests(EmbeddingTestsMixin, unittest.TestCase):
         CONFIG_COMPAT.update({
             'legacy_windows_stdio': 0,
         })
-        # On Windows we run tests with a non-debug build.
-        CONFIG_COMPAT['use_frozen_modules'] = True
+        if not sys.executable.endswith('_d.exe'):
+            # It's not a debug build.
+            CONFIG_COMPAT['use_frozen_modules'] = True
     else:
         config_args = sysconfig.get_config_var('CONFIG_ARGS') or ''
-        if '--with-address-sanitizer' in config_args:
+        if '--with-pydebug' not in config_args:
             CONFIG_COMPAT['use_frozen_modules'] = True
```

Comment:
This code (both platforms) is missing an explanation of what the policy is -- currently "use_frozen_modules defaults to True in debug builds and False otherwise". I was also confused because it's set previously on line 430.

Reply:
I'll take care of that.

Reply:
I ended up making the default "off", no matter what.  In a follow-up PR I'll change the default to "on" for PGO.  At that point I'll expand the comment about the policy.